.gitignore

```diff
@@ -61,5 +61,5 @@ typings/
 .next
 
 application.css
-.sass-cache
+.sass-cache/
 application.css.map
```

Comment:
Add this `.sass-cache/e4a874c94d30a15bed2025ceb1f3cd7342b33966/application.scssc` file to gitignore and run `git rm -rf .sass-cache/` to delete it from git repository